### ISSUE TITLE
Quick and dirty workflow, for now

### ISSUE THREAD

**Issue by deejoe** [Interlock-Rochester/interlock-rochester.github.io#1]
I'm just going to push straight to master while I find my stride on working with this.

If this  horrifies anyone severely, just speak up and we can go more fully more quickly to something more elaborate, like using multiple branches, submitting PRs, getting sign-offs view reviews, etc.

**Comment by deejoe** [comment]
This goes for the github.io preview as well as [the companion src repo](http://github.com/Interlock-Rochester/interlock-rochester.github.io-src)

**Comment by aks4896** [comment]
This will work fine.

**Comment by deejoe** [comment]
On Thu, Mar 09, 2017 at 04:40:28PM -0800, Alex Sojda wrote:
> This will work fine.

OK, nevermind, for now. Looks like we've been doing roughly the same things:
I expanded the README file, made an About page, etc. But when I fetched
prior to pushing I saw your stuff there too and just backed off.

I've switched to the machine that has a key for the server, and have rsync'd
the web site contents into place and modified the web server to serve them. 

So, it may not be quite everything that we want, but for now, it looks
pretty ok.  And with the About entry (which I will probably turn into a
'page' instead of a blog post) it has enough original stuff about us that
its better than having a totally blank website up.
